console-extensions.json

```
@@ -0,0 +1,22 @@
+[
+    {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "netflow-traffic-link",
+      "perspective": "admin",
+      "section": "observe",
+      "name": "Netflow traffic",
```

would be good to add locales, and then call them
```suggestion
      "name": "%plugin_network-observability-plugin~Netflow traffic%",
```

then locales file will need to be named `plugin_network-observability-plugin`

for more info check https://github.com/openshift/console/tree/master/dynamic-demo-plugin